CHANGES.rst

```diff
@@ -25,6 +25,13 @@ vizier
   from ``IndexError`` to ``EmptyResponseError`` [#2980]
 
 
+skyview
+^^^^^^^
+
+- Overlay arguments `lut`, `grid`, and `gridlabel` are removed, as they
```

Comment:
Double backticks are required otherwise sphinx tries to linkify these and fail with an error as there are no API corresponding API references.
```suggestion
- Overlay arguments ``lut``, ``grid``, and ``gridlabel`` are removed, as they
```